hugegraph-mysql/src/main/java/com/baidu/hugegraph/backend/store/mysql/MysqlSessions.java

```diff
@@ -183,43 +183,35 @@ protected String buildExistsTable(String table) {
      * Connect DB without specified database
      */
     protected Connection openWithoutDB(int timeout) {
-        String jdbcUrl = this.config.get(MysqlOptions.JDBC_URL);
-        String url = new URIBuilder().setPath(jdbcUrl)
-                                     .setParameter("socketTimeout",
-                                                   String.valueOf(timeout))
-                                     .toString();
+        String url = this.buildUri(true, timeout);
```

Comment:
the origin logic is autoReconnect=false and no database parameter

Reply:
autoReconnect really do not need, but need useSSL param